test/utils/pickers/describeRangeValidation/testDayViewRangeValidation.tsx

```diff
@@ -70,12 +70,15 @@ export const testDayViewRangeValidation: DescribeRangeValidationTestSuite = (
       describe('with fake timers', () => {
         // TODO: temporary for vitest. Can move to `vi.useFakeTimers`
         let timer: SinonFakeTimers | null = null;
+
         beforeEach(() => {
-          timer = useFakeTimers({ now: new Date(2018, 0, 1), toFake: ['Date'] });
+          timer = useFakeTimers({ now: new Date(2018, 0, 5), toFake: ['Date'] });
```

Comment:
What is the reason behind the day change? 🤔

Reply:
This was intentional. It seems the `1` date wasn't working properly. I didn't look much into it tbh, as I supposed it wasn't a big deal, as it could have been date boundary issues.

Reply:
Gotcha, thanks for noting. 👌